### PR TITLE
Updated Docs for v2.12.0-beta1

### PR DESCRIPTION
**Pull request by JamesKovacs**

### CODE REVIEW

**Review by DmitryLukyanov — APPROVED** [review]

File: Docs/landing/data/releases.toml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 current = "2.11.0"
```
Comment (rstam) [inline Docs/landing/data/releases.toml]: Change all occurrences of 2.11.0 to 2.11.4 in this file.

File: Docs/reference/data/mongodb.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 githubRepo = "mongo-csharp-driver"
 githubBranch = "master"
-currentVersion = "2.11"
+currentVersion = "2.12"
```
Comment (rstam) [inline Docs/reference/data/mongodb.toml]: Shouldn't this remain at 2.11 for now?

**Review by rstam — CHANGES_REQUESTED** [review]

File: Docs/reference/data/mongodb.toml (hunk 2 above)
Comment (JamesKovacs) [reply Docs/reference/data/mongodb.toml]: Probably. Initially I thought it was to build the docs for the new 2.12 branch, but it is probably used to build the landing page. I'll change it and deploy the built docs to my `gh-pages` so we can verify.

**Review by rstam — APPROVED** [review]
LGTM

**Pull request closed without merging**